webapp/src/components/modals/channel_settings/select_channel_subscription.tsx

```diff
@@ -54,6 +57,28 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
             );
         }
 
+        let subName = '';
+        if (subscriptionToDelete && subscriptionToDelete.name) {
+            subName = ` "${subscriptionToDelete.name}"`;
+        }
+
+        let confirmModal = null;
+        if (showConfirmModal) {
+            confirmModal = (
+                <ConfirmModal
+                    cancelButtonText={'Cancel'}
+                    confirmButtonText={'Delete'}
+                    confirmButtonClass={'btn btn-danger'}
+                    hideCancel={false}
+                    message={`Delete Subscription${subName}?`}
+                    onCancel={this.handleDeactivateCancel}
+                    onConfirm={(): void => this.handleConfirmDelete(subscriptionToDelete)}
```

Comment:
```suggestion
                    onConfirm={this.handleConfirmDelete}
```

Then we can make `handleConfirmDelete` use the state variable directly instead of taking a parameter. 0/5 but I think the code is cleaner this way.

Reply:
Gotcha.  I knew to create a state variable but forgot that I no longer needed to pass the sub parameter around any more.